chainerx_cc/chainerx/python/routines.cc

```diff
@@ -275,6 +275,20 @@ void InitChainerxCreation(pybind11::module& m) {
           "endpoint"_a = true,
           "dtype"_a = nullptr,
           "device"_a = nullptr);
+    m.def("tri",
+          [](int64_t n, absl::optional<int64_t> m, int64_t k, py::handle dtype, py::handle device) {
+              if (!m.has_value()) {
+                  m = n;
+              }
```

Comment:
You can skip this condition entirely as the cpp implementation seems to handle it.

Reply:
Okay. I followed https://github.com/chainer/chainer/blob/9be55f275026582f37a13b552ed118aa8b1cbc48/chainerx_cc/chainerx/python/routines.cc#L242

Reply:
Thanks, yes eye should be fixed here too.